distribution.pm

```diff
@@ -191,6 +191,9 @@ sub script_output {
     # if script length is lower there is no point to proceed with more complex solution
     $args{type_command} //= length($script) < 80;
     my $script_path = "/tmp/script$marker.sh";
+    # in some cases client wants validate output of failed script
```

Comment:
Who is the client?, Can you rephrase this?

Reply:
this is API function , usually they called by consumers (or clients) of API . If I will replace "client" with "consumer" it will be ok ? or can you give your version which will suit you for sure ?